packages/node_modules/@webex/plugin-meetings/src/meeting/state.js

```diff
@@ -0,0 +1,150 @@
+
+import StateMachine from 'javascript-state-machine';
+import StateMachineHistory from 'javascript-state-machine/lib/history';
+
+import {MEETING_STATE_MACHINE} from '../constants';
+
+const MeetingStateMachine = {
+  create() {
+    return new StateMachine({
+      init: MEETING_STATE_MACHINE.STATES.IDLE,
+      transitions: [
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.START,
+          from: MEETING_STATE_MACHINE.STATES.IDLE,
+          to: MEETING_STATE_MACHINE.STATES.ON_GOING
+        },
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.START,
+          from: MEETING_STATE_MACHINE.STATES.IDLE,
+          to: MEETING_STATE_MACHINE.STATES.INCOMING
+        },
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.JOIN,
+          from: MEETING_STATE_MACHINE.STATES.IDLE,
+          to: MEETING_STATE_MACHINE.STATES.DIALING
+        },
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.JOIN,
+          from: MEETING_STATE_MACHINE.STATES.ON_GOING,
+          to: MEETING_STATE_MACHINE.STATES.DIALING
+        },
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.JOIN,
+          from: MEETING_STATE_MACHINE.STATES.INCOMING,
+          to: MEETING_STATE_MACHINE.STATES.DIALING
+        },
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.LOCAL,
+          from: MEETING_STATE_MACHINE.STATES.DIALING,
+          to: MEETING_STATE_MACHINE.STATES.CONNECTING_MEDIA_LOCAL
+        },
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.CONNECT,
+          from: MEETING_STATE_MACHINE.STATES.CONNECTING_MEDIA_LOCAL,
+          to: MEETING_STATE_MACHINE.STATES.CONNECTING_MEDIA_REMOTE
+        },
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.ESTABLISH,
+          from: MEETING_STATE_MACHINE.STATES.CONNECTING_MEDIA_REMOTE,
+          to: MEETING_STATE_MACHINE.STATES.ESTABLISHED_MEDIA
+        },
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.LEAVE,
+          from: MEETING_STATE_MACHINE.STATES.ESTABLISHED_MEDIA,
+          to: MEETING_STATE_MACHINE.STATES.TERMINATING
+        },
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.END,
+          from: MEETING_STATE_MACHINE.STATES.TERMINATING,
+          to: MEETING_STATE_MACHINE.STATES.ENDED
+        },
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.CLEAN,
```

Comment:
This should not be part of getStats PR

Reply:
ya it was just additional cleanup to move the meeting state from inside meeting file to a separate meeting state file